torchvision/transforms/functional_tensor.py

```diff
@@ -4,10 +4,8 @@
 
 def vflip(img_tensor):
     """Vertically flip the given the Image Tensor.
-
```

Comment:
Can you revert the removal of newlines introduced in this PR please?

Reply:
@fmassa , I had just removed 1 newline. However, in order to make the format of the docstring of this module look like other ones, I will make the required changes.